src/components/inputs/JuiceSwitch.tsx

```diff
@@ -5,12 +5,13 @@ import { classNames } from 'utils/classNames'
 
 export const JuiceSwitch = ({
   label,
+  description,
   value,
   onChange,
-}: FormItemInput<boolean> & { label?: ReactNode }) => {
-  const switchId = `switch-${label}`
+}: FormItemInput<boolean> & { label?: ReactNode; description?: ReactNode }) => {
+  const switchId = `switch-${label}-${Math.random()}`
```

Comment:
Bug caused when the same component was used as a `label` (e.g. `<TooltipLabel />`)